### PR TITLE
Update getting started in Windows

### PR DESCRIPTION
**Pull request by arushikesarwani94**
Updating Setting up the development environment in Windows documentation to reflect ANDROID_SDK version bump.

React Native is increasing the minimum ANDROID_SDK requirement from Android SDK Platform 31 to Android SDK Platform 33 for 0.72 onwards. Updating our Windows documentation to reflect the same as macOS.

### CODE REVIEW

**Review by cortinico — APPROVED** [review]

File: docs/_getting-started-windows-android.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ To do that, open Android Studio, click on "More Actions" button and select "SDK
 
 Select the "SDK Platforms" tab from within the SDK Manager, then check the box next to "Show Package Details" in the bottom right corner. Look for and expand the `Android 12 (S)` entry, then make sure the following items are checked:
```
Comment (cortinico) [inline docs/_getting-started-windows-android.md:58]: `Android 12 (S)` here is wrong. It should be `Android 13 (Tiramisu)`.
This is also wrong in every other page (win/mac/linux). Are you up for fixing it @arushikesarwani94 ?

Comment (arushikesarwani94) [reply docs/_getting-started-windows-android.md:58]: Ahh missed that. Good catch. Will fix it.

**Comment by arushikesarwani94** [comment]
@cortinico Do we need to backport Windows changes also in the docs for 0.71 ?

**Comment by arushikesarwani94** [comment]
https://github.com/facebook/react-native-website/pull/3617 ✅

**Pull request merged**